test/fixtures.py

```diff
@@ -7,6 +7,7 @@
 
 from os import system, path
 from sys import platform
+#from morphman.common import get_centers, get_path_names, prepare_surface, compute_centerlines
```

Comment:
Mulig å fjerne all utkommentert kode her?

Reply:
Only some of it, it is fixed in the latest commit